### PR TITLE
feat(website): add rubic to ecosystem

### LINKED ISSUE #14584: Rubic
### Title of your project

Rubic 

### Short (1-2 sentence) description of your project

Rubic enhances interoperability through network bridging and cross-chain dev tools for omnichain dApps. Users access diverse assets via varied DEXs under a unified interface with optimized cross-chain transactions.

### Link to your project

https://rubic.exchange/

### Live on Testnet

No

### If your project is live on one of Taiko's testnets, please provide evidence of that here (eg. links, screenshots).

_No response_

### PR DESCRIPTION
**Pull request by 2manslkh**
Closes #14584

@IlyaStor Please reopen an issue when Rubic has been integrated on Taiko testnet. Thank you.

### CODE REVIEW

**Review by dionysuzx — APPROVED** [review]

File: packages/website/components/Ecosystem/EcosystemSection.tsx [hunk 1]
```diff
@@ -261,6 +261,15 @@ const ecosystemData: EcosystemData[] = [
...
     isLive: true,
   },
+  {
+    icon: "/images/ecosystem/rubric.png",
+    name: "Rubric",
+    link: "https://rubic.exchange/",
+    description:
+      "Rubic enhances interoperability through network bridging and cross-chain dev tools for omnichain dApps. Users access diverse assets via varied DEXs under a unified interface with optimized cross-chain transactions.",
+    filters: [],
+    isLive: false,
```
Comment (d1onys1us) [inline packages/website/components/Ecosystem/EcosystemSection.tsx:271]:
```suggestion
    icon: "/images/ecosystem/rubric.png",
    name: "Rubic",
    link: "https://rubic.exchange/",
    description:
      "Rubic enhances interoperability through network bridging and cross-chain dev tools for omnichain dApps. Users access diverse assets via varied DEXs under a unified interface with optimized cross-chain transactions.",
    filters: [],
    isLive: false,
```

Comment (d1onys1us) [reply packages/website/components/Ecosystem/EcosystemSection.tsx:271]: can we also rename the file from `rubric.png` -> `rubic.png`

**Review by dionysuzx — CHANGES_REQUESTED** [review]
look like rubric should be rubic

**Comment by 2manslkh** [comment]
thanks good catch, fixed

**Review by dionysuzx — APPROVED** [review]

**Review by davidtaikocha — APPROVED** [review]

**Pull request merged**